src/NuGet.Clients/NuGet.VisualStudio.Client/registration.json

```diff
@@ -75,6 +75,54 @@
           "title": "@PackageManagement;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}"
         }
       }
+    },
+    "nuGetPackageManager.configurationFiles.externalSettings": {
+      "type": "external",
+      "title": "@117;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}",
+      "backingStoreDescription": "@Text_ConfigurationFiles_Priority;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}",
+
+      "callback": {
+        "packageId": "5fcc8577-4feb-4d04-ad72-d6c629b083cc",
+        "serviceId": "6C09BBE2-4537-48B4-87D8-01BF5EB75901"
+      },
+
+      "realtimeNotifications": true,
+
+      // Settings within this external settings region.  Unlike "normal" settings, these have no registered default values.
+      // Their values are retrieved via IExternalSettingsProvider.GetValueAsync; when the user edits them, the new values are passed to
+      // IExternalSettingsProvider.SetValueAsync.
+      "properties": {
+        "configurationFiles": {
+          "type": "array",
+          "title": "@117;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}",
+          "description": "@Text_ConfigurationFiles_CommonLink;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}",
+
+          // Metadata about items in this array
+          "items": {
+            "type": "object",
+
+            // The properties of items in this array
+            "properties": {
+              "filePath": {
+                "type": "string",
+                "title": "@Text_FilePath_Header;{5fcc8577-4feb-4d04-ad72-d6c629b083cc}"
+              }
+            }
+          },
+
+          // The actual items that appear by default; in most cases, you'd leave this array empty.
+          "default": [
+            {
+            }
+          ],
+          "allowItemEditing": false,
+          "allowAdditionsAndRemovals": false,
+
+          "migrationCallback": {
```

Comment:
migrationCallback doesn't do anything here (since migration isn't an external settings concept) and should be removed.